app/dashboard/templates/dashboard/index.html

```diff
@@ -202,6 +202,13 @@
                 <span class="fa fa-heart" data-toggle="tooltip"
                       title="This alias added to favorite"></span>
               {% endif %}
+
+              {% if alias.hibp_breaches | length > 0 %}
+                <a href="https://haveibeenpwned.com/">
+                  <span class="fa fa-warning text-danger" data-toggle="tooltip"
+                        title="This alias was found in {{ alias.hibp_breaches | length }} data breaches. Check haveibeenpwned.com for more information."></span>
```

Comment:
I wonder if there's a way to create a link to HIBP that shows the data breach for a given email address? It'd save user a step to enter their alias on HIBP :)

Reply:
I've pinged Troy Hunt about this on Twitter. Let's hope it is possible, would be much better yeah :)

Reply:
Yeah, it'd be great!